packages/uui-css/lib/uui-text.css

```diff
@@ -102,6 +102,11 @@
   font-size: var(--uui-size-6);
 }
 
+.uui-text a {
+  color: var(--uui-color-interactive);
+  text-decoration: inherit;
```

Comment:
I would say we need to keep the underline.

The reason for this is as I see this mainly being used within text/paragraphs And in that case, it becomes too hard to distinguish from the rest of the text.